test/JDBC/expected/BABEL-EXTENDEDPROPERTY-v2-vu-verify.out

```diff
@@ -123,11 +123,9 @@ tinyint#!#nvarchar#!#int#!#int#!#varchar#!#sql_variant
 
 EXEC babel_babelfish_extended_properties_proc
 GO
-~~START~~
-int#!#varchar#!#varchar#!#varchar#!#varchar#!#varchar#!#varchar#!#sql_variant
-1#!##!##!##!#DATABASE#!#database property1#!#database property1#!#database property1 before
-1#!##!##!##!#DATABASE#!#database property2#!#database property2#!#database property2 before
-~~END~~
+~~ERROR (Code: 33557097)~~
```

Comment:
This is expected behaviour but not desirable from testing perspective. Aim here is test extended properties and not the permissions. Rather than giving error, we should modify the tests so that could validate all the use cases with required permissions.

Reply:
This testcase was already there in the testfile, I have not added this case. It's just after the changes that this testcase is giving the output error which is expected!